### PR TITLE
Aladdin generated examples

### PR DESCRIPTION
**Pull request by aladdindoc**
- Add Aladdin crafted examples.

### CODE REVIEW

**Comment by yonzhan** [comment]
Aladdin generated examples

File: scripts/ci/aladdin/aladdin.py [hunk 1]
```diff
@@ -50,7 +81,16 @@
...
 examples:
   - name: Show the container registry's identity details. (autogenerated)
-    text: az acr identity show --name mycontainerregistry
+    text: az acr identity show --name MyRegistry
+    crafted: true
+"""
+
+aladdin_helps['acr pack build'] = """
+type: command
+short-summary: Queues a quick build task that builds an app and pushes it into an Azure Container Registry.
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:90]:
```suggestion
short-summary: Queue a quick build task that builds an app and pushes it into an Azure Container Registry.
```

File: scripts/ci/aladdin/aladdin.py [hunk 2]
```diff
@@ -50,7 +81,16 @@
...
+examples:
+  - name: Queues a quick build task that builds an app and pushes it into an Azure Container Registry. (autogenerated)
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:92]:
```suggestion
  - name: Queue a quick build task that builds an app and pushes it into an Azure Container Registry. (autogenerated)
```

File: scripts/ci/aladdin/aladdin.py [hunk 3]
```diff
@@ -531,7 +641,7 @@
 short-summary: Show the details of an Azure Media Services account.
 examples:
   - name: Show the details of an Azure Media Services account. (autogenerated)
-    text: az ams account show --name myazuremediaservicesaccount
+    text: az ams account show --name $amsAccountName --resource-group $resourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:644]:
```suggestion
    text: az ams account show --name MyAmsAccount --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 4]
```diff
@@ -540,7 +650,25 @@
 short-summary: Generate a new client secret for a service principal configured for an Azure Media Services account.
 examples:
   - name: Generate a new client secret for a service principal configured for an Azure Media Services account. (autogenerated)
-    text: az ams account sp reset-credentials --account-name myaccount --resource-group myresourcegroup
+    text: az ams account sp reset-credentials --account-name amsAccount --resource-group resourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:653]:
```suggestion
    text: az ams account sp reset-credentials --account-name MyAmsAccount --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 5]
```diff
@@ -540,7 +650,25 @@
...
+    crafted: true
+"""
+
+aladdin_helps['ams account-filter delete'] = """
+type: command
+short-summary: Delete an account filter.
+examples:
+  - name: Delete an account filter. (autogenerated)
+    text: az ams account-filter delete --account-name myAmsAccount --name myAmsFilter --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:662]:
```suggestion
    text: az ams account-filter delete --account-name MyAmsAccount --name MyAmsFilter --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 6]
```diff
@@ -540,7 +650,25 @@
...
+    crafted: true
+"""
+
+aladdin_helps['ams account-filter list'] = """
+type: command
+short-summary: List all the account filters of an Azure Media Services account.
+examples:
+  - name: List all the account filters of an Azure Media Services account. (autogenerated)
+    text: az ams account-filter list --account-name MyamsAccount --resource-group MyResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:671]:
```suggestion
    text: az ams account-filter list --account-name MyAmsAccount --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 7]
```diff
@@ -549,7 +677,7 @@
 short-summary: Create an asset.
 examples:
   - name: Create an asset. (autogenerated)
-    text: az ams asset create --account-name myaccount --name myasset --resource-group myresourcegroup
+    text: az ams asset create --account-name myAmsAccount --name myAmsName --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:680]:
```suggestion
    text: az ams asset create --account-name MyAmsAccount --name MyAmsName --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 8]
```diff
@@ -558,7 +686,7 @@
 short-summary: Delete an asset.
 examples:
   - name: Delete an asset. (autogenerated)
-    text: az ams asset delete --account-name myaccount --name myasset --resource-group myresourcegroup
+    text: az ams asset delete --account-name myAmsAccount --name myAmsName --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:689]:
```suggestion
    text: az ams asset delete --account-name MyAmsAccount --name MyAmsName --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 9]
```diff
@@ -567,7 +695,7 @@
 short-summary: Lists storage container URLs with shared access signatures (SAS) for uploading and downloading Asset content. The signatures are derived from the storage account keys.
 examples:
   - name: Lists storage container URLs with shared access signatures (SAS) for uploading and downloading Asset content (autogenerated)
-    text: az ams asset get-sas-urls --account-name myaccount --name myasset --permissions Read --resource-group myresourcegroup
+    text: az ams asset get-sas-urls --account-name myAmsAccount --expiry 2020-01-01 --name myAmsAsset --permissions acuw --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:698]:
```suggestion
    text: az ams asset get-sas-urls --account-name MyAmsAccount --expiry 2020-01-01 --name MyAmsAsset --permissions acuw --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 10]
```diff
@@ -585,7 +713,7 @@
 short-summary: Show the details of an asset.
 examples:
   - name: Show the details of an asset. (autogenerated)
-    text: az ams asset show --account-name myaccount --name myasset --resource-group myresourcegroup
+    text: az ams asset show --account-name myAmsAccount --name myAmsName --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:716]:
```suggestion
    text: az ams asset show --account-name MyAmsAccount --name MyAmsName --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 11]
```diff
@@ -603,7 +732,7 @@
 short-summary: Create an asset filter.
 examples:
   - name: Create an asset filter. (autogenerated)
-    text: az ams asset-filter create --account-name amsAccount --asset-name assetName --end-timestamp 200000 --force-end-timestamp true --name filterName --resource-group resourceGroup --start-timestamp 100000 --timescale 1000
+    text: az ams asset-filter create --account-name myamsAccount --asset-name myScaleSet --end-timestamp 2018-12-24 --force-end-timestamp 2018-12-26 --name myScaleSet --resource-group myResourceGroup --start-timestamp 2018-12-26 --timescale 60
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:735]:
```suggestion
    text: az ams asset-filter create --account-name MyAmsAccount --asset-name MyScaleSet --end-timestamp 2018-12-24 --force-end-timestamp 2018-12-26 --name MyScaleSet --resource-group MyResourceGroup --start-timestamp 2018-12-26 --timescale 60
```

File: scripts/ci/aladdin/aladdin.py [hunk 12]
```diff
@@ -612,7 +741,7 @@
 short-summary: Delete a content key policy.
 examples:
   - name: Delete a content key policy. (autogenerated)
-    text: az ams content-key-policy delete --account-name myaccount --name mycontentkeypolicy --resource-group myresourcegroup
+    text: az ams content-key-policy delete --account-name myAmsAccount --name contentKeyPolicyName --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:744]:
```suggestion
    text: az ams content-key-policy delete --account-name MyAmsAccount --name ContentKeyPolicyName --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 13]
```diff
@@ -621,7 +750,7 @@
 short-summary: List all the content key policies within an Azure Media Services account.
 examples:
   - name: List all the content key policies within an Azure Media Services account. (autogenerated)
-    text: az ams content-key-policy list --account-name myaccount --resource-group myresourcegroup
+    text: az ams content-key-policy list --account-name myAmsAccount --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:753]:
```suggestion
    text: az ams content-key-policy list --account-name MyAmsAccount --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 14]
```diff
@@ -630,7 +759,7 @@
 short-summary: Show an existing content key policy.
 examples:
   - name: Show an existing content key policy. (autogenerated)
-    text: az ams content-key-policy show --account-name myaccount --name mycontentkeypolicy --resource-group myresourcegroup
+    text: az ams content-key-policy show --account-name myAmsAccount --name contentKeyPolicyName --resource-group myResourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:762]:
```suggestion
    text: az ams content-key-policy show --account-name MyAmsAccount --name ContentKeyPolicyName --resource-group MyResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 15]
```diff
@@ -639,7 +768,7 @@
 short-summary: Delete a job.
 examples:
   - name: Delete a job. (autogenerated)
-    text: az ams job delete --account-name myaccount --name myjob --resource-group myresourcegroup --transform-name mytransform
+    text: az ams job delete --account-name myAmsAccount --name myFirstJob_007 --resource-group myResourceGroup --transform-name transformName
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:771]:
```suggestion
    text: az ams job delete --account-name MyAmsAccount --name myFirstJob_007 --resource-group MyResourceGroup --transform-name TransformName
```

File: scripts/ci/aladdin/aladdin.py [hunk 16]
```diff
@@ -648,7 +777,16 @@
 short-summary: Show the details of a job.
 examples:
   - name: Show the details of a job. (autogenerated)
-    text: az ams job show --account-name myaccount --name myjob --resource-group myresourcegroup --transform-name mytransform
+    text: az ams job show --account-name myAmsAccount --name myAmsJob --resource-group myResourceGroup --transform-name transformName
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:780]:
```suggestion
    text: az ams job show --account-name MyAmsAccount --name MyAmsJob --resource-group MyResourceGroup --transform-name TransformName
```

File: scripts/ci/aladdin/aladdin.py [hunk 17]
```diff
@@ -648,7 +777,16 @@
...
+    crafted: true
+"""
+
+aladdin_helps['ams job start'] = """
+type: command
+short-summary: Start a job.
+examples:
+  - name: Start a job. (autogenerated)
+    text: az ams job start --account-name myAmsAccount --base-uri 'https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/00000000-b215-4409-80af-529c3e853622/' --files 'Ignite-short.mp4' --name myFirstJob_007 --output-assets $outputAssetName --resource-group $resourceGroup --transform-name $transformName
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:789]:
```suggestion
    text: az ams job start --account-name MyAmsAccount --base-uri 'https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/00000000-b215-4409-80af-529c3e853622/' --files 'Ignite-short.mp4' --name MyFirstJob_007 --output-assets OutputAssetName --resource-group MyResourceGroup --transform-name TransformName
```

File: scripts/ci/aladdin/aladdin.py [hunk 18]
```diff
@@ -657,7 +795,25 @@
 short-summary: Create a live event.
 examples:
   - name: Create a live event. (autogenerated)
-    text: az ams live-event create --account-name myaccount --ips 1.2.3.4/22 5.6.7.8/30 --name myliveevent --resource-group myresourcegroup --streaming-protocol FragmentedMP4
+    text: az ams live-event create --account-name amsAccount --auto-start --ips 1.2.3.4 5.6.7.8/30 --name liveEventName --preview-ips 10.0.0.11 --resource-group resourceGroup --streaming-protocol https://ams.windows.net/54826b22-38d6-4fb2-bad9-b7983a3e853622/
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:798]:
```suggestion
    text: az ams live-event create --account-name AmsAccount --auto-start --ips 1.2.3.4 5.6.7.8/30 --name LiveEventName --preview-ips 10.0.0.11 --resource-group ResourceGroup --streaming-protocol https://ams.windows.net/54826b22-38d6-4fb2-bad9-b7983a3e853622/
```

File: scripts/ci/aladdin/aladdin.py [hunk 19]
```diff
@@ -657,7 +795,25 @@
...
+    crafted: true
+"""
+
+aladdin_helps['ams live-event delete'] = """
+type: command
+short-summary: Delete a live event.
+examples:
+  - name: Delete a live event. (autogenerated)
+    text: az ams live-event delete --account-name amsAccount --name liveEventName --resource-group resourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:807]:
```suggestion
    text: az ams live-event delete --account-name AmsAccount --name LiveEventName --resource-group ResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 20]
```diff
@@ -657,7 +795,25 @@
...
+    crafted: true
+"""
+
+aladdin_helps['ams live-event show'] = """
+type: command
+short-summary: Show the details of a live event.
+examples:
+  - name: Show the details of a live event. (autogenerated)
+    text: az ams live-event show --account-name amsAccount --name liveEventName --resource-group resourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:816]:
```suggestion
    text: az ams live-event show --account-name AmsAccount --name LiveEventName --resource-group ResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 21]
```diff
@@ -666,7 +822,7 @@
 short-summary: Start a live event.
 examples:
   - name: Start a live event. (autogenerated)
-    text: az ams live-event start --account-name myaccount --name myliveevent --resource-group myresourcegroup
+    text: az ams live-event start --account-name amsAccount --name liveEventName --resource-group resourceGroup
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:825]:
```suggestion
    text: az ams live-event start --account-name AmsAccount --name LiveEventName --resource-group ResourceGroup
```

File: scripts/ci/aladdin/aladdin.py [hunk 22]
```diff
@@ -3242,12 +3865,31 @@
...
+    crafted: true
+"""
+
 aladdin_helps['dls account firewall delete'] = """
 type: command
 short-summary: Deletes a firewall rule in a Data Lake Store account.
 examples:
   - name: Deletes a firewall rule in a Data Lake Store account. (autogenerated)
-    text: az dls account firewall delete --account mydatalakestoragegen1 --firewall-rule-name myfirewallrule
+    text: az dls account firewall delete --account mydatalakestoragegen1 --firewall-rule-name myrule
```
Comment (zhoxing-ms) [inline scripts/ci/aladdin/aladdin.py:3892]:
```suggestion
    text: az dls account firewall delete --account MyDatalakeStorageGen1 --firewall-rule-name MyRule
```

**Pull request closed without merging**